.github/workflows/docker.yml

```diff
@@ -38,6 +38,10 @@ on:
         required: false
         default: true
         type: boolean
+    outputs:
+      full-image-name:
+        description: "Full pushed image name including host/registry, name, and tag"
+        value: ${{ jobs.build-docker-image.outputs.full-image-name }}
```

Comment:
does `build-docker-image.outputs.full-image-name` have to be specified in the `build-docker-image` job below too?

Reply:
don't know, just asking

Reply:
It is specified as an output already in that workflow. I just never propagated it up to the main calling workflow.